runtime/drivers/duckdb/transporter/sqlstore_to_duckDB.go

```diff
@@ -76,13 +78,20 @@ func (s *sqlStoreToDuckDB) Transfer(ctx context.Context, srcProps, sinkProps map
 	// TODO :: iteration over fileiterator is similar(apart from no schema changes possible here)
 	// to consuming fileIterator in objectStore_to_duckDB
 	// both can be refactored to follow same path
+	fromAthena := reflect.TypeOf(s.from).AssignableTo(reflect.TypeOf(&athena.Connection{}))
 	for iter.HasNext() {
 		files, err := iter.NextBatch(_sqlStoreIteratorBatchSize)
 		if err != nil {
 			return err
 		}
 
 		format := fileutil.FullExt(files[0])
+		if fromAthena {
+			// Athena doesn't specify ".parquet" extension in output file names
+			// Append ".parquet" extension to the extension generated by Athena
+			format += ".parquet"
+		}
```

Comment:
This reflection really hurts... Is there not a way to have Athena give output files the right file extension with `UNLOAD`?

If not, we should add a way to propagate the format, maybe by adding it in `rillblob.Options` and exposing it as `iter.Format()` or something like that.